components/Comments/Comment.vue

```diff
@@ -170,9 +169,7 @@
         remove: 'comments/remove',
         patch: 'comments/patch'
       }),
-      editorText (newText) {
```

Comment:
As far as I know, the `editorText` method was used to show the confirmation dialog, when a user accidently or intentionally clicked a button to leave the current page. So your latest changes would not be lost. @ionphractal could you confirm?

Actually I would love to see you @Gerald1614 pair-program with a community member. @Gerald1614 did you join our Discord Server already? You can organize a pair-programming with e.g. @ionphractal. He speaks perfect English as far as I know.

Reply:
here's the invitation link to our public Discord Server: https://discord.gg/6ub73U3

Reply:
Hi, I am on discord. i have a very busy week but i will use discord to contact @ionphractal. I will be happy to spend time with him. it will be easier than playign ping pong with chat messages. thanks, Gerald